libraries/core_libs/network/rpc/Test.cpp

```diff
@@ -165,6 +165,41 @@ Json::Value Test::get_account_balance(const Json::Value &param1) {
   return res;
 }
 
+Json::Value Test::get_account_votes(const Json::Value &param1) {
+  Json::Value res;
+  try {
+    if (auto node = full_node_.lock()) {
```

Comment:
This nested if is not necessary - always try not to use nested if's due to readability. CAn be refactored to:
`Json::Value Test::get_account_votes(const Json::Value &param1) {
  Json::Value res;

  auto node = full_node_.lock();
  if (!node) {
    return res;
  }

  try {
    addr_t addr = addr_t(param1["address"].asString());
    uint64_t block_number = param1["block_number"].asUInt64();
    ...`